mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -1104,9 +1104,15 @@ async def add_basic_metrics(
             uids.append(uid)
             model_endpoint_objects_by_uid[uid] = model_endpoint_object
 
-        error_count_res = self.get_error_count(endpoint_ids=uids, get_raw=True)
-        avg_latency_res = self.get_avg_latency(endpoint_ids=uids, get_raw=True)
-        drift_status_res = self.get_drift_status(endpoint_ids=uids, get_raw=True)
+        error_count_res = await run_in_threadpool(
+            self.get_error_count, endpoint_ids=uids, get_raw=True
+        )
+        avg_latency_res = await run_in_threadpool(
+            self.get_avg_latency, endpoint_ids=uids, get_raw=True
+        )
+        drift_status_res = await run_in_threadpool(
+            self.get_drift_status, endpoint_ids=uids, get_raw=True
+        )
```

Comment:
why not move it to a function and run that function only? saving the overhead of moving those 3 altogether to a thread?